lib/mapbox/isochrone.rb

```diff
@@ -0,0 +1,39 @@
+module Mapbox
+  class Isochrone
+    include Mapbox::APIOperations::Request
+    extend Mapbox::HashUtils
+
+    def self.assemble_params(options={})
+      contours_minutes = options[:contours_minutes]
+      contours_colors = options[:contours_colors]
+      polygons = options[:polygons]
+      denoise = options[:denoise]
+      generalize = options[:generalize]
```

Comment:
@Bathlamos they are optional - still can be used, though.

Reply:
Hm, I don't think so @Bathlamos.

`isochrone = Mapbox::Isochrone.isochrone("walking", "-118.22258,33.99038", {contours_minutes: [5,10,15], polygons: false})` returns a LineString, while `isochrone = Mapbox::Isochrone.isochrone("walking", "-118.22258,33.99038", {contours_minutes: [5,10,15], polygons: true})` returns a Polygon. Are you seeing otherwise?

Reply:
That's because they are encoded at line 15 of the file, not because of the variables at lines 8, 9 and 10. But it's really a minor point 😄

Reply:
@Bathlamos I totally read your comment wrong 😂that makes sense. I'll remove those unused variables and merge. Thank you!